src/elf_message/elf_message_field.c

```diff
@@ -0,0 +1,33 @@
+#include "global.h"
+#include "z64elf_message.h"
+
+ElfMessage gOverworldNaviMsgs[] = {
+/* 0000   0 */ ELF_MESSAGE_FLAG(CHECK, 0x0140, false, 0x05), /* eventChkInf[0] & 0x20 */
+/* 0004   1 */ ELF_MESSAGE_FLAG(CHECK, 0x0141, false, 0x09), /* eventChkInf[0] & 0x200 */
+/* 0008   2 */ ELF_MESSAGE_FLAG(CHECK, 0x0142, false, 0x12), /* eventChkInf[1] & 0x4 */
+/* 000C   3 */ ELF_MESSAGE_FLAG(CHECK, 0x0143, false, 0x14), /* eventChkInf[1] & 0x10 */
+/* 0010   4 */ ELF_MESSAGE_FLAG(CHECK, 0x0144, false, 0x40), /* eventChkInf[4] & 0x1 */
+/* 0014   5 */ ELF_MESSAGE_SONG(CHECK, 0x0145, false, ITEM_SONG_SARIA),
+/* 0018   6 */ ELF_MESSAGE_STRENGTH_UPG(CHECK, 0x0146, true, 0), // TODO goron bracelet
```

Comment:
Since we don't know yet if we're even going to have an enum or defines for upgrade values, I personally think we should omit the TODOs here for now